src/constants/deck_constants.asm

```diff
@@ -65,5 +65,8 @@ ENDM
 	deck_const FLAMETHROWER_DECK           ; $34
 	deck_const RESHUFFLE_DECK              ; $35
 	deck_const IMAKUNI_DECK                ; $36
-DEF DECKS_END EQU const_value - 1
-DEF DECK_IDS_END EQU DECKS_END - 2
+DEF NUM_VALID_DECKS EQU const_value - 1
+DEF NUM_DECK_IDS EQU const_value - 2
```

Comment:
It was a bit intentional that these constants were 2 apart from each other, since that's the difference between a `_DECK` constant and a `_DECK_ID` constant. I suppose it doesn't have to be that way though..

Reply:
`NUM_DECK_IDS` is now one larger than the old `DECK_IDS_END`.
What about making `NUM_VALID_DECKS` one larger than the original `DECKS_END`?
(`const_value` instead of `const_value - 1`)